javascript-data-model/order-history.js

```diff
@@ -0,0 +1,46 @@
+var orderHistory = [
+  {
+    orderedDate: '2020-08-04',
+    totalPrice: 34,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    deliveredDate: '2020-08-08',
+    orderTitle: 'JavaScript for impatient programmers',
```

Comment:
so the update we want to make to your data model is to create an `items` property which will contain an array as its value and then have the data for each item in their own object: 

```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```